### DISCUSSION TITLE
How to draw an S-bend with specific width

### DISCUSSION THREAD

**Discussion by YuleiDing** [gdsfactory/gdsfactory#1028] [Q&A] [answered]
Hello, everyone!
I often use "components.bend_S" to connect my waveguides, it's very convenient.
But several days ago, I found that I could't change the width of bend_S. Whatever do I set the width in the bracket, the width of bend_S is always 0.5um.  

In the picture below, you can see this problem.
I want a bend_S with a width of 2 um, so I set the width equal to 2um. But in the picture, the width of bend_S that I got still equals to 0.5um. So how can I solve this problem?

![20221220211115](https://user-images.githubusercontent.com/105275217/208677422-2306b455-6164-43c2-bae8-12a495822fd6.png)

**Comment by joamatab** [comment, marked as answer]
Thank you Yule for reporting the issue

I fixed the bug here
https://github.com/gdsfactory/gdsfactory/pull/1029

if you update to gdsfactory 6.14.1 it should be fixed
